web/src/App.js

```diff
@@ -60,75 +52,79 @@ function App() {
       }
 
       // FIXME: use the status_id from the event
-      if (payload.event == "StatusChanged") {
+      if (payload.event === "StatusChanged") {
         loadInstallation(dispatch);
       }
 
-      if (payload.event == "Progress") {
+      if (payload.event === "Progress") {
         updateProgress(dispatch, payload.progress);
       }
     });
   }, []);
 
-  const isInstalling = installation.status != 0;
+  const isInstalling = installation.status !== 0;
   const { progress } = installation;
 
   return (
-    <ChakraProvider theme={theme}>
-        <Box minH="100vh" p={3}>
-          <Flex>
-            <Spacer />
-            <ColorModeSwitcher />
-          </Flex>
-          <VStack spacing={8}>
-            <Heading as="h1">Welcome to $INSTALLER-80</Heading>
-            <Divider />
-
-            <Category title="Language" icon={Languages}>
-              <LanguageSelector
-                value={l10n.language}
-                options={l10n.languages}
-                onChange={(language) => setOptions({ language }, dispatch)} />
-            </Category>
-
-            <Category title="Target" icon={HardDrive}>
-              <TargetSelector
-                value={storage.disk || "Select a device"}
-                options={storage.disks}
-                onChange={disk => setOptions({ disk }, dispatch)}
-              />
-              <Proposal data={storage.proposal}/>
-            </Category>
-
-            <Category title="Product" icon={Archive}>
-              <ProductSelector
-                value={software.product || "Select a product"}
-                options={software.products}
-                onChange={(product) => setOptions({ product }, dispatch)}
-              />
+    <>
+      <Stack hasGutter>
+        <StackItem>
+          <TextContent>
+            <Text component={TextVariants.h1}>Welcome to D-Installer</Text>
+          </TextContent>
+        </StackItem>
+
+        <StackItem>
+          <Category title="Language" icon={LanguagesSelectionIcon}>
+            <LanguageSelector
+              value={l10n.language || "Select language"}
```

Comment:
:warning: NOTE that we should start applying https://suse.eosdesignsystem.com/writing/ux-writing here. Looks to me that it'd help to have a clear and concise UI. I.e., better UX.

Reply:
I mean, not only there. In general.